### DISCUSSION TITLE
lazer: some settings dont persist a client restart

### DISCUSSION THREAD

**Discussion by aplotor** [ppy/osu#28549] [Q&A]
- in lazer, i noticed these settings dont persist a client restart::
  1. changed user online status (always goes back to online status when reopened)
  2. osu!direct search filters (always goes back to default)

not sure if those are intended. but it would be nice to persist those settings

thank you

**Comment by Stoppedpuma** [comment]
> osu!direct search filters (always goes back to default)

Intentional for now, tracked at #22009
